README.md

```diff
@@ -141,6 +141,35 @@ You can set the default behavior for VPA creation using some flags. When specifi
 * `--include-namespaces` - create VPAs in these namespaces, in addition to any that are labeled
 * `--exclude-namespaces` - when `--on-by-default` is set, exclude this comma-separated list of namespaces
 
+#### Enable Namespaces
+
+Namespaces are considered enabled or managed by goldilocks when the Namespace
+has the enabled label set to "true", for example:
+
+```
+kubectl label ns goldilocks goldilocks.fairwinds.com/enabled=true
+```
+
+#### VPA Update Mode
```

Comment:
My reasoning here behind adding these docs is because the Namespace will already have the `vpa-update-mode` label visible and users may try fiddling with this without knowing how it works. At least with the docs we can explain what that label is, what it does, and that it shouldn't really be used.